pkg/buildkit/buildkit.go

```diff
@@ -18,10 +18,18 @@ import (
 	"github.com/moby/buildkit/util/progress/progressui"
 	"github.com/pkg/errors"
 	"golang.org/x/sync/errgroup"
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/dominodatalab/hephaestus/pkg/buildkit/archive"
 )
 
+var clientCheckBackoff = wait.Backoff{ // retries after 10ms 50ms 250ms 1250ms 6250ms
```

Comment:
seems quite fast relative to the time it might take.
does it really mater if its 10ms, 50ms 250ms, vs the more normal 1s, 2s , 4s...

Reply:
We running a test we found that under light load that it took < 1s but > 250ms. I think I'm just trying to be as fast as possible here but you're probably right. Maybe something like 500ms, 1s, 2s, 4s... seems more reasonable?

Reply:
good compromise